include/deal.II/lac/block_vector_base.h

```diff
@@ -975,6 +964,15 @@ class BlockVectorBase : public Subscriptor
   void update_ghost_values () const;
 
   /**
+   * A variable that indicates whether this vector supports distributed data
+   * storage.
+   *
+   * For the current class, the variable equals the value declared for the
+   * type of the individual blocks.
+   */
+  bool supports_distributed_data() const;
```

Comment:
@Rombur Never mind. I now see what you try to achieve and how you want to use ```supports_distributed_data``` - I agree that a static member function will be awkward to use in case of inheritance and working with base classes.